index.d.ts

```diff
@@ -230,6 +230,19 @@ declare module '@primer/components' {
   export const Sticky: React.FunctionComponent<PositionComponentProps>
   export const Fixed: React.FunctionComponent<PositionComponentProps>
 
+  export interface SideNavProps extends CommonProps, BorderProps, Omit<React.HTMLAttributes<HTMLDivElement>, 'color'> {
+    bordered?: boolean
```

Comment:
@emplums We could also make a border the default and since this has `BORDER` props now consumers could just `border="none"` 🤔

Reply:
Actually this makes the nesting thing harder so I retract the idea